File: Lesson6 Simple-Intermediate Math/Problem1/Solutions/Solution.java
import java.util.Scanner
public class Solution{
    public static void main(String[]args){
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt(); 
        int freq[] = new int[11];
        int max = 100;
        for(int i = 0; i < a; i++){
            int x = sc.nextInt();
            freq[x]++;
            max = Math.max(max, freq[x]);
        }
        for(int i = 0; i < freq.length(); i++){
            if(freq[i] == max){
                System.out.println(i);
                break;
            }
        }
        sc.close()
    }
}

File: Lesson6 Simple-Intermediate Math/Problem1/Solutions/Solution.cpp
#include <iostream>

using namespace std;

int main()
{
   int a; cin>>a;
   int freq[11] = {0}; int MAX = 0;
   for(int i = 0; i < a; i++){
       int x; cin>>x;
       freq[x]++;
       MAX = max(MAX, freq[x]);
   }
   for(int i = 0; i < 11; i++){
       if(freq[i] == MAX){
           cout<<i<<endl;
           break;
       }
   }
   return 0;
}
